### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by czarnecki**

### CODE REVIEW

File: src/main/java/de/adesso/kicker/match/controller/MatchController.java [hunk 1]
```diff
@@ -67,11 +67,14 @@ public ModelAndView postAddMatch(@Valid Match match, BindingResult bindingResult
     }
 
     private ModelAndView defaultAddMatchView(ModelAndView modelAndView) {
-        modelAndView.addObject("match", new Match());
-        modelAndView.addObject("users", userService.getAllUsers());
-        modelAndView.addObject("currentUser", userService.getLoggedInUser());
-        modelAndView.addObject("notifications",
-                notificationService.getNotificationsByReceiver(userService.getLoggedInUser()));
+        var match = new Match();
```
Comment (Flo4l) [inline src/main/java/de/adesso/kicker/match/controller/MatchController.java]: Is this a better way to program?
looks a bit more readable but also less performant

Comment (JanSchneider1) [inline src/main/java/de/adesso/kicker/match/controller/MatchController.java]: In my opinion, I would call the method on 'new Match()' and 'userService.getAllUsers()' directly as it looks a lot more cluttered that way and also it 'hides' the reason why you're using variables for the other one (which is fine).

Comment (czarnecki) [reply src/main/java/de/adesso/kicker/match/controller/MatchController.java]: I removed one unnecessary database call. This should be more performant than before.

Comment (Flo4l) [reply src/main/java/de/adesso/kicker/match/controller/MatchController.java]: i mea that every object gets saved into a new variable before it is passed to the model

Comment (czarnecki) [reply src/main/java/de/adesso/kicker/match/controller/MatchController.java]: The compiler will optimise that most likely so it doesn't really matter.

**Pull request merged**